Tests/RequestKitTests/TestHelper.swift

```diff
@@ -26,4 +26,8 @@ internal class Helper {
         let data = try! Data(contentsOf: URL(fileURLWithPath: path))
         return try! JSONDecoder().decode(T.self, from: data)
     }
+
+    public static func getNSError(from error: Error?) -> NSError? {
+        return (error as? NSError)
```

Comment:
Any clue what's up with Conditional downcast from 'Error?' to 'NSError' is a bridging conversion; did you mean to use 'as'?? I get the warning on Xcode and swift test on macOS but no warnings on Linux.

Reply:
It fails on Linux with an compiler error. Don't know if this was because the compiler thought this is still Swift 3. Would love to just use `error as NSError` but seems to be unavailable on Linux.

Reply:
@pietbrauer The fix for this is currently on the Swift master, until it is released folks from the Swift Forums have suggested using platform checks.

Reply:
Implemented on 01ae897